.setup/INSTALL_SUBMITTY_HELPER_SITE.sh

```diff
@@ -149,8 +153,4 @@ chmod 550 ${SUBMITTY_INSTALL_DIR}/site/cgi-bin/git-http-backend
 # cache needs to be writable
 find ${SUBMITTY_INSTALL_DIR}/site/cache -type d -exec chmod u+w {} \;
 
-# return the course index page (only necessary when 'clean' option is used)
```

Comment:
This is removed as a continuing part of https://github.com/Submitty/Submitty/pull/4337